### PR TITLE
6 get /api/reviews/review id/comments

### PR DESCRIPTION
**Pull request by CharlieH94**

### CODE REVIEW

File: sandbox.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-SELECT * FROM reviews WHERE review_id = 1;
\ No newline at end of file
+-- SELECT reviews.owner, reviews.title, reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, reviews.designer, CAST(COUNT(comment_id) AS INTEGER) AS comment_count
+--     FROM reviews
+--     LEFT JOIN comments ON comments.review_id = reviews.review_id
+-- GROUP BY reviews.review_id
+-- ORDER BY reviews.created_at DESC;
+
+
+SELECT * FROM comments WHERE review_id = 2 ORDER BY created_at DESC;
```
Comment (Slyose) [inline sandbox.sql]: Seems like a good idea to have this file in the .gitignore :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -109,6 +109,39 @@ describe("app", () => {
     });
   });
 
+  describe("GET /api/reviews/:review_id/comments", () => {
```
Comment (Slyose) [inline __tests__/app.test.js:112]: Great testing so far, but also needs error handling to be tested:
A 404 for an id with no reviews
a 400 for an invalid id e.g /api/reviews/**egg**/comments
And finally, a 200 for a review with no comments, that should respond with an empty array of comments

**Review by Slyose — CHANGES_REQUESTED** [review]

File: sandbox.sql (hunk 1 above)
Comment (CharlieH94) [reply sandbox.sql]: I thought I had but obviously didn't do it right ha

**Review by HannahBurks — APPROVED** [review]
Hey Charlie. That is fine, you can merge that now. Nice work! 😄  The reason you have two different things for ID is if an ID does not exist it should throw a 404 but if it is the case the ID exists, there is  just no comments made yet, it should give an empty array :)

**Comment by cjplatten** [comment]
All good

**Pull request merged**